legend-engine-xt-relationalStore-trino-execution-tests/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/connection/tests/api/dynamicTestConnections/TrinoTestContainers.java

```diff
@@ -0,0 +1,101 @@
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.plan.execution.stores.relational.connection.tests.api.dynamicTestConnections;
+
+import org.finos.legend.engine.plan.execution.stores.relational.connection.tests.api.DynamicTestConnection;
+import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.store.relational.connection.DatabaseType;
+import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.store.relational.connection.RelationalDatabaseConnection;
+import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.store.relational.connection.authentication.DelegatedKerberosAuthenticationStrategy;
+import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.store.relational.connection.specification.TrinoDatasourceSpecification;
+import org.finos.legend.engine.shared.core.vault.PropertiesVaultImplementation;
+import org.finos.legend.engine.shared.core.vault.Vault;
+import org.finos.legend.engine.shared.core.vault.VaultImplementation;
+import org.testcontainers.containers.TrinoContainer;
+import org.testcontainers.containers.startupcheck.MinimumDurationRunningStartupCheckStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+import java.time.Duration;
+import java.util.Properties;
+
+public class TrinoTestContainers
+        implements DynamicTestConnection
+{
+    @Override
+    public DatabaseType getDatabaseType()
+    {
+        return DatabaseType.Trino;
+    }
+
+    public TrinoContainer trinoContainer = new TrinoContainer(DockerImageName.parse("trinodb/trino"))
+            .withStartupCheckStrategy(new MinimumDurationRunningStartupCheckStrategy(Duration.ofSeconds(10)));
+
+    private VaultImplementation vaultImplementation;
+
+    @Override
+    public void setup()
+    {
+        this.startTrinoContainer();
+        this.registerVault();
+    }
+
+    private void startTrinoContainer()
+    {
+        System.out.println("Starting setup of dynamic connection for database: Trino ");
+
+        long start = System.currentTimeMillis();
+        this.trinoContainer.start();
+        String containerHost = this.trinoContainer.getHost();
+        int containerPort = this.trinoContainer.getMappedPort(8080);
+        long end = System.currentTimeMillis();
+
+        System.out.println("Completed setup of dynamic connection for database: Trino on host:" + containerHost + " and port:" + containerPort + " , time taken(ms):" + (end - start));
+    }
+
+    public void registerVault()
+    {
+        Properties properties = new Properties();
+        properties.put("sqlServerAccount.user", "SA");
```

Comment:
Can these properties be removed ? Looks like copy/paste from the sqlserver code.

Reply:
This is removed